calibration_utils.py

```diff
@@ -88,9 +89,10 @@ def polygon_from_image_name(image_name):
 class StereoCalibration(object):
     """Class to Calculate Calibration and Rectify a Stereo Camera."""
 
-    def __init__(self, traceLevel, outputScaleFactor):
+    def __init__(self, traceLevel: Optional[float] = 1.0, outputScaleFactor: Optional[float] = 0.5, disableCamera: Optional[list] = []):
```

Comment:
So that you dont need to call or specify in main function, where you are using this submodule. For example, depthai/calibrate.py needs all three of them, while FactoryCalibration does not need them.